src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/PhaseRecordComponentTest.kt

```diff
@@ -0,0 +1,116 @@
+package uk.gov.justice.digital.hmpps.managerecallsapi.component
+
+import com.natpryce.hamkrest.assertion.assertThat
+import com.natpryce.hamkrest.equalTo
+import org.junit.jupiter.api.Test
+import org.springframework.http.HttpStatus.BAD_REQUEST
+import uk.gov.justice.digital.hmpps.managerecallsapi.controller.BookRecallRequest
+import uk.gov.justice.digital.hmpps.managerecallsapi.controller.Phase
+import uk.gov.justice.digital.hmpps.managerecallsapi.domain.CroNumber
+import uk.gov.justice.digital.hmpps.managerecallsapi.domain.FirstName
+import uk.gov.justice.digital.hmpps.managerecallsapi.domain.LastName
+import uk.gov.justice.digital.hmpps.managerecallsapi.random.randomNoms
+import java.time.LocalDate
+import java.time.OffsetDateTime
+
+class PhaseRecordComponentTest : ComponentTestBase() {
+  private val nomsNumber = randomNoms()
+  private val bookRecallRequest = BookRecallRequest(
+    nomsNumber,
+    FirstName("Barrie"),
+    null,
+    LastName("Badger"),
+    CroNumber("1234/56A"),
+    LocalDate.now()
+  )
+
+  @Test
+  fun `start the BOOK Phase for recall returns BAD_REQUEST`() {
+    val recall = authenticatedClient.bookRecall(bookRecallRequest)
+
+    authenticatedClient.startPhase(recall.recallId, Phase.BOOK, BAD_REQUEST)
+  }
+
+  @Test
+  fun `start the Phase for recall creates new record`() {
+    val phase = Phase.ASSESS
+    val recall = authenticatedClient.bookRecall(bookRecallRequest)
+
+    val record = authenticatedClient.startPhase(recall.recallId, phase)
+
+    assertThat(record.recallId(), equalTo(recall.recallId))
+    assertThat(record.phase, equalTo(phase))
+    assertThat(record.startedByUserId(), equalTo(authenticatedClient.userId))
+    assertThat(record.startedDateTime, !equalTo<OffsetDateTime>(null))
+    assertThat(record.endedByUserId, equalTo(null))
+    assertThat(record.endedDateTime, equalTo(null))
+  }
+
+  @Test
+  fun `starting the phase twice for a recall overwrites first record`() {
```

Comment:
"starting the phase" makes me expect the phase to be identified; maybe "the" -> "a" for these test titles?